src/main/ast-to-doc.js

```diff
@@ -35,6 +35,9 @@ function printAstToDoc(ast, options, alignmentSize = 0) {
   const cache = new Map();
   const path = new AstPath(ast);
 
+  const embeds = new Map();
+  await printEmbeddedLanguages(path, mainPrint, options, printAstToDoc, embeds);
```

Comment:
`const embeds = await printEmbeddedLanguages(...) `?

Reply:
Won't work. `embed` is used inside `mainPrint`, which is called inside `printEmbeddedLanguages`.